public/login.css

```diff
@@ -55,13 +55,19 @@ button:hover {
   border: 1px solid #8e0000;
 }
 
+.message-info {
+  background: #8bc34a;
+  border: 1px solid #5a9216;
+}
+
 .message:empty {
   display: none;
 }
 
 /* stylelint-disable-next-line selector-max-id */
 #login-form {
   position: relative;
+  background-color: #fff;
```

Comment:
If you render an iframe inside of something you can adjust inputs to get full window width and and render the iframe inside of a `Card`. I would look better.